### DISCUSSION TITLE
Quero aumentar a quantidade de requisições a API do Github

### DISCUSSION THREAD

**Discussion by EricksonFerreira** [community/community#133038] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Aparece o seguinte erro:

Limite de taxa de API excedido para 149.100.155.60. (Mas aqui estão as boas notícias: solicitações autenticadas têm um limite de taxa maior. Confira a documentação para mais detalhes.) Se você entrar em contato com o Suporte do GitHub para obter ajuda, inclua o ID da solicitação CD08:15A86F:DD304:195267:669DE7C9 e o registro de data e hora 2024-07-22 05:02:01 UTC.

Queria orientações como faço para aumentar a taxa de limite de requisições

**Comment by davevad93** [comment, marked as answer]
Hi @EricksonFerreira ,

If you make unauthenticated requests, you can make up to 60 requests per hour, to increase the limit you have to make authenticated requests with a `Personal Access Token`, which allows up to 5000 per hour.  For more info see [Rate limits for the REST API](https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28) from the docs.

**Comment by ghostinhershell** [comment]
Hi @EricksonFerreira, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
